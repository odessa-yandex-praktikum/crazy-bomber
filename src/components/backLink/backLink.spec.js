import React from 'react';
import renderer from 'react-test-renderer';
import {BackLink} from './index';

it('renders correctly', () => {
    const tree = renderer.create(<BackLink />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly on error page', () => {
    const tree = renderer.create(<BackLink customClass="error-page__back-link" />).toJSON();
    expect(tree).toMatchSnapshot();
});
