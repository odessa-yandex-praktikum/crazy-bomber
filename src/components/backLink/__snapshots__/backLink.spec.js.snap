// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="backlink"
  onClick={[Function]}
>
  &lt;&lt;
</button>
`;

exports[`renders correctly on error page 1`] = `
<button
  className="backlink error-page__back-link"
  onClick={[Function]}
>
  &lt;&lt;
</button>
`;
